### ISSUE TITLE
Huge battery drain

### ISSUE THREAD

**Issue by nholtappels** [rpetrich/ProSwitcher#367]
This might be more of a backgrounder than a ProSwitcher issue, but since I'm not sure what causes it, I'll post it here.
Since I began using PS my battery lasts less than half as long as it did w/o backgrounder and PS. After removing both, the battery drain disappeared.
Hoping that it might have been some strange bug I DFU restored my iPad and rejailbroke it. I used it a couple of days w/o backgrounder in order to find out if there was anything else jailbreak-related draining my battery. There wasn't.
Now that I've reinstalled BG and PS, the battery drain is back.
I'm going to remove them now, but I hope that I'll be able to use PS again soon.
Keep up the good work!
PS: I'd take ProSwitcher over iOS-backgrounding any day

**Comment by grp** [comment]
There is no such thing as "batter drain". Using more than one app at a time, of course, affects the battery.

**Comment by nholtappels** [comment]
Don't get me wrong, I love ProSwitcher and I really appreciate your work.
I do realize that running multiple apps leads to a higher battery consumption.
If I had noticed only a slight difference I would never have filed this bug.
I only did so because I've been using ProSwitcher on all of my iPhones (1, 3G, 4) for quite a long time now, and it's never effected my battery life much. Maybe shortened it by an hour or so..
Anyway.. If you think this is expected behavior, it apparently is.
I'll still wait for an official release to try my luck
